test/e2e/framework/framework_constants.go

```diff
@@ -193,15 +194,17 @@ const (
 	Config                      = "config"
 	TanzuCLIE2ETestBinaryPath   = "TANZU_CLI_E2E_TEST_BINARY_PATH"
 	WiredMockHTTPServerStartCmd = "docker run --rm -d -p 8080:8080 -p 8443:8443 --name %s -v %s:/home/wiremock rodolpheche/wiremock:2.25.1"
-	HttpMockServerStopCmd       = "docker container stop %s"
-	HttpMockServerName          = "wiremock"
+	HTTPMockServerStopCmd       = "docker container stop %s"
+	HTTPMockServerName          = "wiremock"
 	defaultTimeout              = 5 * time.Second
 
 	TMCEndpointForPlugins        = "/v1alpha1/system/binaries/plugins"
 	TMCMockServerEndpoint        = "http://localhost:8080"
 	TMCPluginsMockServerEndpoint = "http://localhost:8080/v1alpha1/system/binaries/plugins"
-	HttpContentType              = "application/json; charset=utf-8"
+	HTTPContentType              = "application/json; charset=utf-8"
 
 	// k8s CRD file
 	K8SCRDFilePath = "../../framework/config/cli.tanzu.vmware.com_cliplugins.yaml"
+
+	DefaultInputConfigDataFilePath = "test/e2e/framework/config/input_data.yaml"
```

Comment:
This does not seem like something that belongs under `Framework`. This is specific to the Tanzu CLI Core tests and should not be part of the Framework code.

Also, Do we really need `DefaultInputConfigDataFilePath`? Can we configure `TANZU_CLI_E2E_INPUT_CONFIG_DATA_FILE_PATH` in the `Makefile` directory from where we invoke the e2e test?